docs-src/basic_usage.rst

```diff
@@ -251,6 +251,11 @@ Modals use the same blocks that compose messages with the addition of an `input`
 
 See `views.open <https://api.slack.com/methods/views.open>`_ more details and additional parameters.
 
+Also, to run the above example, the following `Slack app configurations <https://api.slack.com/apps>`_ are required.
+
+* Add a slash command named ``/open-modal`` with a valid Request URL: ``https://{your-public-domain}/slack/events``
```

Comment:
Optional, but would it be possible to use a global shortcut instead of a slash command?

Reply:
Indeed, that's much better! I will do.

Reply:
done